src/style.css

```diff
@@ -1,3 +1,101 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background-color: #e0e0e0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 50px;
+  background-color: #fff;
+  width: 1100px;
+  height: 40px;
```

Comment:
you don't need to hardcode this values, also see 1'st point in checklist about header height